core/domain/feedback_services.py

```diff
@@ -475,7 +694,12 @@ def _send_batch_emails(recipient_list, feedback_message_reference):
 
 def _send_instant_emails(
         recipient_list, feedback_message_reference):
-    """send feedback message email instantly."""
+    """send feedback message email instantly.
+
+    Args:
+        recipient_list: list of str. A list of recipient id
```

Comment:
Can you also make the docstrings clearer? See the examples I gave in the previous function.